xbmc/epg/EpgDatabase.cpp

```diff
@@ -99,6 +105,17 @@ void CEpgDatabase::UpdateTables(int iVersion)
 
   if (iVersion < 9)
     m_pDS->exec("ALTER TABLE epgtags ADD sIconPath varchar(255);");
+  
+  if (iVersion < 10)
```

Comment:
This breaks updating of the Epg database from version 9 to version 10 with SQLite. I get the following error message:
```
SQL: SQL error or missing database
Query: ALTER TABLE epgtags ADD sOriginalTitle  varchar(128), sCast           varchar(255), sDirector       varchar(255), sWriter         varchar(255), iYear           integer, sIMDBNumber     varchar(50);
```